float32 max_vel_forward
float32 max_vel_backward
float32 max_vel_theta
float32 acc_lim_x
float32 acc_lim_theta
float32 turning_radius
float32 wheelbase
float32 goal_tolerance_xy
float32 goal_tolerance_yaw
float32 min_obstacle_distance
float32 obstacle_inflation_radius
float32 dynamic_obstacle_inflation_radius
float32 dt_ref
float32 dt_hysteresis
bool include_dynamic_obstacles
bool include_costmap_obstacles
bool oscillation_recovery
bool allow_init_with_backward_motion
bool save_settings